python/lsst/ci/prepare.py

```diff
@@ -28,6 +28,21 @@
 from future.utils import with_metaclass
 
 
+class RemoteError(Exception):
```

Comment:
I think this needs a docstring to generally say under what conditions it may be raised.

Reply:
Aye.